docs/configs.md

```diff
@@ -1,54 +0,0 @@
-# Configurations
```

Comment:
Some additional doc is probably needed somewhere in order to explain that we now have a `setup` command.
Some examples may need to change too. Maybe in the README.md